scan/src/components/TypeID.re

```diff
@@ -0,0 +1,57 @@
+type pos_t =
+  | Title
+  | Subtitle
+  | Text;
+
+let fontSize =
+  fun
+  | Title => Text.Xxl
+  | Subtitle => Text.Lg
+  | Text => Text.Md;
+
+let lineHeight =
+  fun
+  | Title => Text.Px(23)
+  | Subtitle => Text.Px(18)
+  | Text => Text.Px(16);
+
+let letterSpacing =
```

Comment:
Why do we have this function? Seems useless.

Reply:
You're right.